src/slider.test.events.multiple.ts

```diff
@@ -0,0 +1,896 @@
+import { assert, aTimeout, expect, fixture, html } from '@open-wc/testing';
+import sinon from 'sinon';
+import { sendKeys } from '@web/test-runner-commands';
+import Slider from './slider.js';
+
+it('dispatches an "input" event typing in the minimum input', async () => {
+  const host = await fixture<Slider>(
+    html`<glide-core-slider label="Label" multiple></glide-core-slider>`,
+  );
+
+  const spy = sinon.spy();
+  host.addEventListener('input', spy);
+
+  const minimumInput = host.shadowRoot?.querySelector<HTMLInputElement>(
+    '[data-test="minimum-input"]',
+  );
+
+  minimumInput?.focus();
+
+  await sendKeys({ type: '1' });
+
+  expect(spy.callCount).to.equal(1);
+});
+
+it('dispatches an "input" event typing in the maximum input', async () => {
+  const host = await fixture<Slider>(
+    html`<glide-core-slider label="Label" multiple></glide-core-slider>`,
+  );
+
+  const spy = sinon.spy();
+  host.addEventListener('input', spy);
+
+  const maximumInput = host.shadowRoot?.querySelector<HTMLInputElement>(
+    '[data-test="maximum-input"]',
+  );
+
+  maximumInput?.focus();
+
+  await sendKeys({ type: '1' });
+
+  expect(spy.callCount).to.equal(1);
+});
+
+it('dispatches a "change" event blurring the minimum input', async () => {
+  const host = await fixture<Slider>(
+    html`<glide-core-slider label="Label" multiple></glide-core-slider>`,
+  );
+
+  const spy = sinon.spy();
+  host.addEventListener('change', spy);
+
+  const minimumInput = host.shadowRoot?.querySelector<HTMLInputElement>(
+    '[data-test="minimum-input"]',
+  );
+
+  minimumInput?.focus();
```

Comment:
Tab to and away from instead of programmatically blurring? There are a few instances of this.

Reply:
I dig it - updating! I found myself reaching for `focus()` and `blur()` more for convenience, but tabbing to and away is more common in our code base. Thanks for the eyes!